src/om_pouch/core.cljs

```diff
@@ -118,8 +118,10 @@
           query (get-in pouchput [:params])]
       
       (let [id (:_id query)
-            nitem {:item/by-id {id {:_id id :item "I JUST CHANGED YOU"}}}]
-        (cb nitem pouchput))))
+            stable-id (random-uuid)
+            result {'pouchput {:tempids {[:item/by-id id] [:item/by-id stable-id]}}
+                    [:item/by-id stable-id] {:_id stable-id :item (str (:item query) " - I JUST CHANGED YOU")}}]
```

Comment:
This is the general shape of a response from a remote that adds novelty.  Merging novelty using idents (the `[:item/by-id id]` keys) will not require a read query passed to the cb fn.